### PR TITLE
[Main] Update peoplehr test cases

### PR DESCRIPTION
**Pull request by SanduDS**
## Automation tests
 - Unit tests 
   >  Done
 - Integration tests
   > Done

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: peoplehr/tests/test.bal [hunk 1]
```diff
@@ -24,7 +24,10 @@ configurable string baseURL = os:getEnv("BASE_URL");
 configurable string emailAddress = os:getEnv("EMAIL_ADDRESS");
 configurable string password = os:getEnv("PASSWORD");
 configurable string queryName = os:getEnv("QUERY_NAME");
-string employeeId = "test01";
+configurable int screenId = 380477;
+configurable string employeeId = "test01";
+int txnId = 0;
+int applicantId = 7696154;
```
Comment (abeykoon) [inline peoplehr/tests/test.bal]: Should not we made these configurable as well?

File: peoplehr/tests/test.bal [hunk 2]
```diff
@@ -98,8 +100,10 @@ function testCreateNewEmployee() returns error? {
         Location: "Colombo03",
         Department: "Engineering"
     };
-    OperationStatus response = check baseClient->createNewEmployee(employee);
-    test:assertFalse(response.isError, msg = response.Message);
+    OperationStatus|error response = baseClient->createNewEmployee(employee);
+    if response is error {
+        test:assertFail(response.message());
```
Comment (abeykoon) [inline peoplehr/tests/test.bal:111]: We need to log the response from server if the assert fails. Then only we can know what is the error server sent. We can do that by

```
test:test:assertFail(xxx, msg = response.Message);
```

File: peoplehr/tests/test.bal [hunk 3]
```diff
@@ -244,89 +254,86 @@ function testGetEmployeeScreenDetail() returns error? {
...
 
-@test:Config {enable: true}
+@test:Config {enable: true, dependsOn: [testAddNewCustomScreenTransaction]}
 function testGetEmployeeScreenDetailByEmployeeID() returns error? {
     log:printInfo("Get employee screen detail by employee ID");
     ScreenDetailByEmployeeIDRequest payload = {
-        ScreenId: 378661,
-        EmployeeId: "PW1"
+        ScreenId: screenId,
+        EmployeeId: "PW10"
```
Comment (abeykoon) [inline peoplehr/tests/test.bal]: `PW10 ` is a magic number. Shall we make it configurable or add a constant?

File: peoplehr/tests/test.bal [hunk 4]
```diff
@@ -348,8 +355,8 @@ function testGetAppraisalDetailsByEmployeeID() returns error? {
 function testGetAppraisalDetailsByAppraisalID() returns error? {
     log:printInfo("Get appraisal details by appraisal ID");
     AppraisalDetailsByAppraisalIDRequest payload = {
-        EmployeeId: "123",
-        AppraisalId: 0
+        EmployeeId: "PW10",
+        AppraisalId: 1
```
Comment (abeykoon) [inline peoplehr/tests/test.bal]: `AppraisalId ` why it is `1` not `2`? 
Shall we make it a configurable or a constant?

File: peoplehr/tests/test.bal (hunk 1 above)
Comment (SanduDS) [reply peoplehr/tests/test.bal]: I guess no need as these are used by the test cases and initialized themselves.

File: peoplehr/tests/test.bal (hunk 2 above)
Comment (SanduDS) [reply peoplehr/tests/test.bal:111]: `test:assertFail` takes only string msg as the input and, if an error occurs, it shows the message.
image

Comment (abeykoon) [reply peoplehr/tests/test.bal:111]: +1. Ack

**Review by abeykoon — APPROVED** [review]

**Pull request merged**